Source/scrollrt.cpp

```diff
@@ -893,8 +893,11 @@ void DrawView(int StartX, int StartY)
 	} else if (questlog) {
 		DrawQuestLog();
 	}
-	if (!chrflag && plr[myplr]._pStatPts != 0 && !spselflag
-		&& (!questlog || SCREEN_HEIGHT >= SPANEL_HEIGHT + PANEL_HEIGHT + 49 || SCREEN_WIDTH >= 4 * 320)) {
+#if SCREEN_HEIGHT-PANEL_HEIGHT-SPANEL_HEIGHT >= 74 || PANEL_X > SPANEL_WIDTH

```

Comment:
Pre processor statements will make it harder to implement run time resolutions as they will have to be rewritten. Also you want PANEL_LEFT as _X is in relation to the back buffer and not the screen

Reply:
But this is optimized way. Performance depends on this procedure.
When you make resolution switching support, you change this code.

Reply:
Performance optimizing when there isn't a performance issue is known as premature optimization. It leads to a complicated code that is hard to maintain. It isn't too long ago that I cleaned up a lot of this from the original code https://github.com/diasurgical/devilutionX/pull/361. I don't plan to merge code that adds more code that we know will have to be changed any way. Please make the requested changes for this PR to be ready for merge.

Reply:
P.s. the compiler would optimize away a comparison of static numbers so there actually din't any performance difference between the two solutions.

Reply:
I understand you, but I think differently.
I close the request, as I see all my commits are displayed at you and you can merge them.

Reply:
Ok, but we will only accept PRs that follow the project standard. The simple reason is that there isn't much point in swapping one issue for another.